### DISCUSSION TITLE
Example to use context in visibility API

### DISCUSSION THREAD

**Discussion by rmanibus** [fomantic/Fomantic-UI#1041] [Behaviors (Api, Form Validation, Visibility)] [answered]
# Help Wanted

## Problem
Hi,
I can't figure out how to use the context property in the visibility API.
I tried the following :
```
  $('#fixedDiv')
    .visibility({
      type   : 'fixed',
      context: '#containerDiv',
    })
  ;
```
But nothing happens. It work correctly without the visibility property but I want the element to be fixed only inside the container.
What am I missing here ? Is it possible to add some example in the documentation ?

Thank you

**Comment by y0hami** [comment, marked as answer]
This JSFiddle may help you understand its usage https://jsfiddle.net/jqLmg3oz/

**Comment by rmanibus** [comment]
So it is not designed to fix content only when a specific div is passing on screen. I used sticky instead.

Thank you very much.
